server/handlers/handlers.go

```diff
@@ -247,10 +247,14 @@ func (ah *ActivityHandler) handleCreatedActivity(activityIds msteams.ActivityIds
 	post, errorFound := ah.msgToPost(channelID, senderID, msg, chat, false)
 	ah.plugin.GetAPI().LogDebug("Post generated")
 
-	newPost, appErr := ah.plugin.GetAPI().CreatePost(post)
-	if appErr != nil {
-		ah.plugin.GetAPI().LogError("Unable to create post", "Error", appErr)
-		return
+	var newPost *model.Post
+	if post.Message != "" || len(post.FileIds) > 0 {
```

Comment:
we are not creating a post here if the message is empty, but what if we later update that post from MS Teams to edit the post with some text?

Reply:
Yes, you're right. We should not include this change in the plugin.